src/private/templates/assigner.latte

```diff
@@ -127,4 +131,31 @@
 
 {block footerbottom}
     {$tplutils::includeScript("js/assigner.js", true)}
+    <script>
+        function startCooldownTimer(duration) {
+            var timer = duration, minutes, seconds;
+            var cooldownElement = document.getElementById('cooldown-timer');
+            if (!cooldownElement) return;
+
+            var interval = setInterval(function () {
```

Comment:
Please update the text immediately after page loads (currently it's updated on first interval run - 1 second after page load)